Gruntfile.coffee

```diff
@@ -75,6 +78,7 @@ module.exports = (grunt) ->
         files:
           'dist/droplet-full.js': ['./src/main.coffee']
         options:
+          ignore: ignoredLanguages
```

Comment:
Yeah I *think* I'm doing this the right way.  Coffeescript doesn't have `var` statements, and `if` statements don't create a new scope.  The generated JavaScript will look something like:
```javascript
var ignoredLanguages;

if (process.env.LANGUAGE) {
  ignoredLanguages = ["./src/languages/!(" + process.env.LANGUAGE + ").coffee"];
}
```